Original_CCJ/h_common.cpp

```diff
@@ -109,6 +109,7 @@ brack_type *h_pop (brack_stack *st)
 
 
 double ccj(char *sequence, char *structure){
+    create_size_penalties(strlen(sequence));
```

Comment:
@IanWark, @HosnaJabbari: is there a stronger bound on the size in penalty_by_size (I think there is a constant bound on the size of interior loops...)

In particular, if there is a constant bound, there is no need to construct the size_penalties_class on the heap